src/sentry/static/sentry/app/components/charts/pieChart.jsx

```diff
@@ -19,19 +78,63 @@ class PieChart extends React.Component {
 
     // Note, we only take the first series unit!
     const [firstSeries] = series;
+    const seriesPercentages = this.getSeriesPercentages(firstSeries);
 
     return (
       <BaseChart
+        ref={this.chart}
+        onChartReady={this.handleChartReady}
+        onEvents={{
+          // when legend highlights it does NOT pass dataIndex :(
+          highlight: ({name, ...args}) => {
```

Comment:
Do you need `...args`? You don't use it.

Reply:
👍 Same with `mouseover` function below